witness/src/Dscp/Snowdrop/AccountValidation.hs

```diff
@@ -152,34 +165,22 @@ assertExists thing message = do
       Just it -> do
         return it
 
--- TODO: Fix and unmess when fee will be actually required.
--- accountComputeFee
---   :: forall ctx
---   .  StateTx Ids Proofs Values
---   -> ERoComp Exceptions Ids Values ctx Integer
--- accountComputeFee = \StateTx{..} -> do
---     let ty = StateTxType $ getId (Proxy @AccountTxTypeId) AccountTxTypeId
---     author
---         :: Author
---         <- requirePart txProof (ProofProjectionFailed ty)
---     before <- assertExists (auAuthorId author) AuthorDoesNotExist
---     let updates = Map.toList $ changeSet txBody
---     let accountChanges = updates <&> \pair@(k, _) -> (k, proj pair)
---     changes <- mapM checkThatAccountIsUpdatedOnly accountChanges
---     let (payer, recipients) = List.partition (is (auAuthorId author)) changes
---     (paid, deltas) <-
---         (,) <$> validateSaneDeparture payer before
---             <*> mapM validateSaneArrival recipients
---     let difference = paid - sum deltas
---     () <- validateIff SumMustBeNonNegative (difference >= 0)
---     return difference
+-- TODO: this and '{A,a}uthenticate' should be in Dscp.SD.Util
+canAffordFees
+    :: forall ctx
+    .  Account
+    -> Word32
+    -> ERoComp Exceptions Ids Values ctx ()
+canAffordFees account size = do
+    validateIff CannotAffordFees $ aBalance account >= fromIntegral size
```

Comment:
Shouldn't it be `aBalance account >= size + txInValue`?

Reply:
No, this is for account state with already retracted payment.

Reply:
Thus we can use it for transactions w/o payment.